morevna-ep4.msrt

```diff
@@ -181,7 +181,7 @@
 37
 00:05:05,280 --> 00:05:07,600
 [rus] Прощай, добрый молодец...
-[eng] Good-Bye, young man...
+[eng] Farewell, young man...
```

Comment:
"young man" sounds rather formal and something an "older gentleman" would say, so I matched it with the slightly more formal/older "farewell". If this was just an unintended consequence of the translation process and the character was supposed to be more informal/younger/have a "cool guy" kind of persona (kind of the impression I get from looks alone), I might go with something short and quick like "Bye kid". Some of his other lines might need revision to make his characterization consistent throughout.